gaia-fast-list.js

```diff
@@ -312,6 +317,9 @@ var GaiaFastListProto = {
         z-index: 100;
 
         margin: 0 !important;
+        margin-top: -0.5px !important;
+        padding-top: 0.5px;
+        width: calc(100% + 1px);
```

Comment:
This stops the sticker headers from being glitchy when layerized `<img>`s fade in underneath them.